tgui/packages/tgui/interfaces/RoboticsControlConsole.js

```diff
@@ -0,0 +1,166 @@
+import { Fragment } from 'inferno';
+import { act } from '../byond';
+import { Box, Button, NoticeBox, Section, Tabs, LabeledList } from '../components';
+
+export const RoboticsControlConsole = props => {
+  const { state } = props;
+  const { config, data } = state;
+  const { ref } = config;
```

Comment:
Just pointing out that it's possible to use `useBackend` and forward the state at the same time.
```js
  const { state } = props;
  const { act, data } = useBackend(props);
```

Reply:
Thanks, was not sure about that. Replaced.